common/views/components/Divider/Divider.tsx

```diff
@@ -3,49 +3,26 @@ import styled from 'styled-components';
 import { PaletteColor } from '@weco/common/views/themes/config';
 
 type Props = {
-  color: PaletteColor;
-  isStub?: boolean;
-  isKeyline?: boolean;
-  isThin?: boolean;
+  color?: PaletteColor;
+  variant?: 'stub' | 'thick';
 };
 
 const Rule = styled.hr<Props>`
   text-align: left;
   border: 0;
-
+  height: 1px;
   ${props =>
-    props.color &&
-    `
-    background-color: ${props.theme.color(props.color)};
-  `}
+    props.color && `background-color: ${props.theme.color(props.color)};`};
 
-  ${props =>
-    props.isStub &&
-    `
-    width: 60px;
-    height: 5px;
-  `}
+  // Variants
+  ${props => props.variant === 'stub' && 'width: 60px; height: 5px;'}
 
-  ${props =>
-    props.isKeyline &&
-    `
-    height: 1px;
-  `}
-
-  ${props =>
-    props.isThin &&
-    `
-    height: 2px;
-  `}
+  ${props => props.variant === 'thick' && 'height: 2px;'}
```

Comment:
Previously called `thin`, but it was thicker than the default component. This variant actually isn't used anywhere so I might remove it completely...